addon/-private/helpers.js

```diff
@@ -309,3 +313,37 @@ export function getProperty(object, pathToProp) {
 }
 
 export const assign = Ember.assign || Ember.merge;
+
+/**
+ * @public
+ *
+ * Fills an input, textarea, or contenteditable element with the passed-in content.
+ *
+ * @param {$Element} $element             jQuery DOM Element
+ * @param {string} content                Content to be inserted into the element
+ * @param {Object} options                Options for error reporting in the case
+ * where this helper is used within an execution context
+ * @param {string} options.selector       Element selector
+ * @param {Ceibo} options.pageObjectNode  PageObject node containing the method
+ * which, when invoked, resulted in this call to `fillElement`
+ * @param {string} options.pageObjectKey  Key of method on PageObject
+ * which, when invoked, resulted in this call to `fillElement`
+ * @return
+ *
+ * @throws Will throw an error if called on a contenteditable element that has
+ * `contenteditable="false"`
+ */
```

Comment:
I would move it to a new helper file inside the execution context folder so we avoid people using it on any property directly `/addon/-private/execution_context/helper.js`.

The helpers in this file are used all over the place but for this one is better to avoid it's widespread use.

> Is it correct to mark it @public because it is imported by the execution contexts

Mark it as @private please. 

> Is this level of documentation right for this kind of helper? 

It's not that necessary at this level of helpers but it's nice to have.

> does it need its own unit test?

I think that by testing `fillable` and `selectable` is enough to ensure that this function is working properly. On the other hand if there are some scenarios you think are important to cover with unit tests, feel free to add those, but I think we're good as it is right now.